src/components/orderDetails.js

```diff
@@ -114,6 +171,8 @@ const mapStateToProps = (state) => ({
     loading: state.createReturn.loading,
     error: state.createReturn.error,
     order: state.createReturn.order,
+    quantityToReturn: state.belReducer.quantityToReturn,
+    quantityToExchange: state.belReducer.quantityToExchange
```

Comment:
Fine for now but you'll eventually want to move things out of belReducer and/or rename belReducer.